src/cpp/session/modules/clang/RCompilationDatabase.cpp

```diff
@@ -461,7 +461,7 @@ namespace {
 
 FilePath compilationConfigFilePath()
 {
-   return module_context::scopedScratchPath().completePath("cpp-complilation-config");
+   return module_context::scopedScratchPath().completePath("cpp-complilation-config-v2");
```

Comment:
Idea: maybe instead of adding v2 we can make the new folder name unique by spelling `compilation` correctly? :-)

Reply:
Hah! Good idea :-)